content/en/blog/_posts/2023-07-24-unknown-version-interoperability-proxy-alpha.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: blog
+title: "Unknown Version Interoperability Proxy Alpha"
+date: 2023-05-24
```

Comment:
Please use the date that the release comms team have assigned.

/hold
pending that being used

Reply:
Hi @sftim , thanks, where can I get the date that I need to use here?

Reply:
@Rishit-dagli can you help me with what date should I use here?

Reply:
Ah, I see they already commented on this PR.